src/webhook/mutation/pod/dataingest/volumes_test.go

```diff
@@ -0,0 +1,30 @@
+package dataingest_mutation
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestSetupVolumes(t *testing.T) {
+	t.Run("should add dataingest volumes", func(t *testing.T) {
+		pod := &corev1.Pod{}
+
+		setupVolumes(pod)
+
+		require.Len(t, pod.Spec.Volumes, 2)
```

Comment:
just asserting the length seems like it could lead to issues

Reply:
then we would have the exact same logic in the test, I would like to avoid that